tweets/221021-long-haul.tweet

```diff
@@ -0,0 +1,9 @@
+#EthereumClassic has the fundamentals of what a blockchain should be. 
+
+Many will soon find out. 
+
+We are early.... and here for the long haul!
+
+---
```

Comment:
it doesn't need --- I think

Reply:
This is what separates the two tweets in the thread